wiki/Beatmapping/Timing/en.md

```diff
@@ -0,0 +1,63 @@
+
+The [beatmap editor](/wiki/Client/Beatmap_editor) provides [timing settings](/wiki/Client/Beatmap_editor/Timing) with a [ruler-like timeline view](/wiki/Client/Beatmap_editor/Timelines#timing) for temporal placement of [hit objects](/wiki/Gameplay/Hit_object). When objects are placed, they are [snapped](/wiki/Beatmapping/Snapping) to timeline ticks, i.e. subdivisions of a [measure](/wiki/Music_theory/Measure), allowing objects to easily line up with notes of the song.
+
+Additionally, the timing of a beatmap controls the following features:
+
+- Heartbeat of the [osu! cookie](/wiki/Client/Interface/Cookie) in the main menu and [song select screen](/wiki/Client/Interface#song-select) (as well as other menu effects that follow the beat)
+- Pulsation of hit objects during [kiai time](/wiki/Gameplay/Kiai_time)
+- Drum track added by the [Nightcore](/wiki/Gameplay/Game_modifier/Nightcore) mod
+- Appearance of bar lines in [osu!taiko](/wiki/Game_mode/osu!taiko) and [osu!mania](/wiki/Game_mode/osu!mania)
+
+Timing partially reflects *properties of a specific audio file* and may change once the audio is replaced or modified in a way that affects its waveform. It is recommended to ensure that the audio file follows the [ranking criteria](/wiki/Ranking_Criteria#audio) before establishing timing.
+
+## Song structure
+
+*Main article: [Music theory](/wiki/Music_theory)*
+
+Almost every song can be divided into sections of different tempo, and the goal of timing is to divide up a beatmap to match those sections. Understanding a few concepts in music theory may help when discussing this process:
+
+- A [beat](/wiki/Music_theory/Beat) is the base unit of rhythm, usually corresponding to the quarter note. Other notes are expressed as its fractions and multiples.
+- A [measure](/wiki/Music_theory/Measure), or a bar, is a fixed amount of beats repeating throughout the song. The first beat of a measure is emphasised and called a [downbeat](/wiki/Music_theory/Downbeat).
+- A [time signature](/wiki/Music_theory/Time_signature) specifies how many beats fit in a single measure.
+- [Rhythm](/wiki/Music_theory/Rhythm) is the pattern of sounds and silences a song.
+- [Tempo](/wiki/Music_theory/Tempo) is the speed of the song, measured in beats per minute (BPM).
+
+## Timing process
+
+*Main article: [How to time songs](/wiki/Guides/How_to_Time_Songs)*\
+*See also: [Beatmap editor/Timing](/wiki/Client/Beatmap_editor/Timing)*
+
+### Single BPM
+
+A beatmap's first [offset](/wiki/Beatmapping/Offset) ([uninherited timing point](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point), i.e. "red line") is the amount of time between the beginning of the audio file and the first downbeat, and the starting point used by osu! to calculate positions of other beats. A single timing point is often enough for songs that are digitally produced or edited seamlessly, having no BPM or time signature changes, or metronome resets<!-- TODO: link -->.
```

Comment:
gah, what if you have an uninherited timing point before the first downbeat (for pickups notes)

maybe something like this would be better?

```suggestion
The amount of time between the beginning of the audio file and the first downbeat, known as the beatmap's "offset", is typically the place for the first [uninherited timing point](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point), the starting point used by osu! to calculate positions of other beats. One of these timing points is often enough for songs that are digitally produced or edited seamlessly, having no BPM or time signature changes, or metronome resets<!-- TODO: link -->.
```